src/applications/disability-benefits/686c-674/config/chapters/veteran-information/veteran-information/VeteranInformationComponent.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+import { connect } from 'react-redux';
+import moment from 'moment';
+
+const VeteranInformationComponent = ({
+  user: {
+    gender,
+    dob,
+    userFullName: { first, last },
+  },
+}) => {
+  let dateOfBirthFormatted = '-';
+  let genderFull = '-';
+  if (dob) {
+    dateOfBirthFormatted = moment(dob).format('MMMM Do YYYY');
+  }
+  if (gender === 'M') {
+    genderFull = 'Male';
+  } else if (gender === 'F') {
+    genderFull = 'Female';
+  }
+  const alertContent = (
+    <dl className="vads-u-margin--0 vads-u-padding-left--2">
+      <dt className="vads-u-line-height--4 vads-u-padding-bottom--2">
+        <strong>
+          {first} {last}
+        </strong>
+      </dt>
+      <dd className="vads-u-line-height--4 vads-u-padding-bottom--2">
+        Date of birth: {dateOfBirthFormatted}
+      </dd>
+      <dd className="vads-u-line-height--4">Gender: {genderFull}</dd>
+    </dl>
+  );
+  return (
+    <div>
+      <p>This is the information we have on file for you.</p>
+      <AlertBox
+        className="vads-u-padding--0 vads-u-background-color--white vads-u-border-color--primary usa-alert-info::before"
```

Comment:
Couple thoughts:
1. You can actually just get rid of the last class on here, `usa-alert-info::before` since in your custom scss I noticed you're just targeting `.usa-alert` directly. 
1. It might be a slight nit, but the left border width is still a little big. If you open the [Mockups](https://xd.adobe.com/view/ca825c11-2268-458f-4fec-6aaecfec5b2d-bcb3/screen/dc153448-40b1-4d35-b890-698dca690305/Add-Modify-Dependents-Veteran-Details) and click on the `</>` icon in the top right, it'll actually open the markup view where you can see all of the values for color, padding, margin, etc. For this particular piece, it looks like the mockups suggest 6px. We could get close to that using a utility class like `vads-u-border-left--5px` or `vads-u-border-left--7px`, or, adding another custom css class.

Reply:
Totally agree, I updated this to 5px. Also, nice find on inspecting the styles in the mockup!!